flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBValueState.java

```diff
@@ -64,7 +64,7 @@
 	 * @param stateDesc The state identifier for the state. This contains name
 	 *                           and can create a default state value.
 	 */
-	RocksDBValueState(ColumnFamilyHandle columnFamily,
+	public RocksDBValueState(ColumnFamilyHandle columnFamily,
```

Comment:
The reason why I made the state constructors public so if you sublcass the RocksDBStatebackend you can still manually instantiate them. You wouldnt be able to call the protected constructor from your package though. 

It's no biggie, I can change it if you still think its better as protected.